docs/getting_started/installation.md

```diff
@@ -73,4 +73,10 @@ Run it with maximum number of cores:
     singularity run -e khanlab_hippunfold_latest.sif \
     PATH_TO_BIDS_DIR PATH_TO_OUTPUT_DIR participant -p --cores all
 
-Note that you may need to adjust your [Singularity options](https://sylabs.io/guides/3.1/user-guide/cli/singularity_run.html) to ensure this container can read and write to yout input and output directories, repsectively. 
+Note that you may need to adjust your [Singularity options](https://sylabs.io/guides/3.1/user-guide/cli/singularity_run.html) to ensure this container can read and write to yout input and output directories, repsectively. For example, if your home directory is full or inaccessible, you may wish to set the following singularity parameters:
+
+    export SINGULARITY_CACHEDIR=/YOURDIR/.cache/singularity
+    export SINGULARITY_BINDPATH=/YOURDIR:/YOURDIR
+    export HIPPUNFOLD_CACHE_DIR=/YOURDIR/.cache/hippunfold/
```

Comment:
This line shouldn't be needed here, since the instructions here are for using the hippunfold container, which already has the models downloaded (and HIPPUNFOLD_CACHE_DIR set)

Reply:
agreed and removed